### PR TITLE
support cloudwatch data source in grafana

### PR DESCRIPTION
**Pull request by m-richo**
as discussed in issue #12 

grafana cloudwatch data source is looking for aws credentials in `$HOME/.aws/credentials`.
this PR creates the credentials file in `/var/vcap/jobs/grafana/home/.aws/credentials` containing AWS keys and set $HOME env-var for the grafana-server process launched by the `chpst` command.

I've tested it locally in bosh-lite and works fine.

### CODE REVIEW

File: jobs/grafana/templates/credentials [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[default]
```
Comment (vito) [inline jobs/grafana/templates/credentials:1]: Looks like this template isn't referenced by the spec. It should be put under `home/.aws/...` right?

**Review by vito — CHANGES_REQUESTED** [review]

File: jobs/grafana/templates/credentials (hunk 1 above)
Comment (m-richo) [reply jobs/grafana/templates/credentials:1]: doh.  I'm a dill.  You are right.  
i went to take in your most recent changes before submitting the PR and managed to remove it from the spec file.  I'll update, rebase and send through.

**Review by vito — APPROVED** [review]

File: jobs/grafana/templates/credentials [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+aws_access_key_id = <%= p("grafana.aws.aws_access_key_id") %>
```
Comment (vito) [inline jobs/grafana/templates/credentials:2]: Ah these should also have a default or something; right now this template will blow up rendering if those properties aren't specified.

**Review by vito — CHANGES_REQUESTED** [review]

File: jobs/grafana/templates/credentials (hunk 2 above)
Comment (m-richo) [reply jobs/grafana/templates/credentials:2]: any preference on having them as empty strings vs dummy keys?

Comment (vito) [reply jobs/grafana/templates/credentials:2]: Empty string should be fine as long as it parses. Not sure if it does in this format.

Comment (m-richo) [reply jobs/grafana/templates/credentials:2]: tested it with defaults of empty string.  Bosh creates file successfully with

```
$ cat credentials
[default]
aws_access_key_id =
aws_secret_access_key =
region =
```

**Comment by m-richo** [comment]
G'day,
I've rebased to include the grafana 4.0 updates and it still seems to work fine.  Do you have any further feedback or questions?

Cheers,
Michael

**Review by vito — APPROVED** [review]

**Comment by vito** [comment]
thanks and sorry for the delay!

**Pull request merged**